spotler/backend/plugin/src/main/kotlin/com/ritense/spotler/plugin/SpotlerPlugin.kt

```diff
@@ -0,0 +1,35 @@
+package com.ritense.spotler.plugin
+
+import com.ritense.plugin.annotation.Plugin
+import com.ritense.plugin.annotation.PluginAction
+import com.ritense.plugin.annotation.PluginProperty
+import com.ritense.plugin.domain.ActivityType
+import org.camunda.bpm.engine.delegate.DelegateExecution
+
+@Plugin(
+    key = "spotler",
+    title = "Spotler Plugin",
+    description = "This plugin allows you to send emails from within a process"
+)
+class SpotlerPlugin {
+
+    @PluginProperty(key = "clientId", secret = false)
```

Comment:
secret false is dat niet de default? zo ja dan mag het weg.

Reply:
Nee het heeft geen default, moet gespecificeerd worden